contracts/crypto-verify/src/msg.rs

```diff
@@ -82,6 +82,24 @@ pub enum QueryMsg {
     /// No pagination - this is a short list.
     #[returns(ListVerificationsResponse)]
     ListVerificationSchemes {},
+    /// Webauthn component verification
+    #[returns(VerifyResponse)]
+    VerifyWebauthn {
+        /// Authenticator data
+        authenticator_data: Binary,
+        /// Client data (JSON encoded)
+        client_data_json: String,
+        /// Challenge value
+        challenge: Binary,
+        /// X coordinate of public key point
+        x: Binary,
+        /// Y coordinate of public key point
+        y: Binary,
+        /// r component of signature
+        r: Binary,
+        /// s component of signature
+        s: Binary,
```

Comment:
For those 4 integers, could you add how they are encoded in binary?

Reply:
Unfortunately. I'm not sure what part of it is so expensive though.. I'm inclined to say it's the SHA256 digest since inside of WASM it has to fall back to its software implementation.
But I have to properly bench that

Reply:
Added documentation. Basically they are just big-endian representations of the integers. Nothing fancy.